### ISSUE TITLE
OPTIONS verb missing

### ISSUE THREAD

**Issue by luansens** [FastEndpoints/FastEndpoints#671]
I need allow OPTIONS verb on my endpoints, but i don`t see one way to do it on documentation and the Http method enum does not have this value.

How I can allow this method for my endpoints, also if possible allow for all endpoints on global configuration.

**Comment by dj-nitehawk** [comment]
it's not possible to add an http verb globally. but you can set any string you want like below:

```cs
sealed class MyEndpoint : EndpointWithoutRequest
{
    public override void Configure()
    {
        Routes("my-endpoint");
        Verbs("OPTIONS", "GET");
        AllowAnonymous();
    }
}
```
is the OPTIONS verb needed for doing manual CORS or something?

**Comment by luansens** [comment]
need set this verb for all endpoints configured, because axios always use this verb to prefligth and I don't like to disable it on axios, the way I would like to solve this is allow OPTIONS verb for all endpoints. Or so in group configuration something like
```
public class Auth : Group
{
    public Auth()
    {
        Configure("auth", ep =>
        {
           ep.Verbs.Add("OPTIONS")
        });
    }
}
```

**Comment by dj-nitehawk** [comment]
implemented in `v5.25.0.1-beta`
the following is now possible:
```csharp
//global configurator
app.UseFastEndpoints(
       c => c.Endpoints.Configurator =
                ep =>
                {
                    ep.AdditionalVerbs(Http.OPTIONS, Http.HEAD);
                })
    
//endpoint group
sealed class SomeGroup : Group
{
    public SomeGroup()
    {
        Configure(
            "prefix",
            ep =>
            {
                ep.AdditionalVerbs(Http.OPTIONS, Http.HEAD);
            });
    }
}
```
